msteams-platform/bots/how-to/conversations/conversation-basics.md

```diff
@@ -5,38 +5,45 @@ ms.topic: overview
 ms.author: anclear
 keyword: conversations basics receive message send message picture message channel data adaptive cards
 ---
+
 # Conversation basics
 
 [!INCLUDE [pre-release-label](~/includes/v4-to-v3-pointer-bots.md)]
 
-A conversation is a series of messages sent between your bot and one or more users. There are three types of conversations, also called scopes in Teams:
+A conversation is a series of messages sent between your Microsoft Teams bot and one or more users. There are three types of conversations, also called scopes in Teams:
 
 | Conversation type | Description |
 | ------- | ----------- |
-|  `teams` | Also called channel conversations, visible to all members of the channel. |
-| `personal` | Conversations between bots and a single user. |
-| `groupChat` | Chat between a bot and two or more users. Also enables your bot in meeting chats. |
+| `channel` | This conversation type is also called channel conversations, visible to all members of the channel. |
```

Comment:
This is repetitive.

Reply:
I did not understand what is repetitive. Please elaborate.